### DISCUSSION TITLE
How good is it in terms of parseability?

### DISCUSSION THREAD

**Discussion by fernandocamargoai** [rendercv/rendercv#32] [Q&A]
I've been using awesome-cv for years, but have been noticing that some websites struggle to parse the contents of some work history parts of the resume. Like, the first few are parsed correctly, but the latter ones might not be parsed correctly. So, I guess it affects me when ATS can't parse it very well. So, I just found this project and was wondering how good the final PDF is in terms of being parsed. I'll give it a try later today or this week and comment back here, but I'd also like to hear other people's experiences.

**Comment by sinaatalay** [comment]
Thank you for opening this discussion!

I didn’t test it, so I don’t know. If someone knows how to do this test, it would be very nice. We can update the PDF if needed.

**Reply by fernandocamargoai** [reply]
In general, it's about uploading the CV to Indeed or any other website that parses it and lets you edit the experiences. With awesome-cv, for example, I had the first few work experiences being parsed correctly, but then there were a few that the role name would be empty for an entry and then it would appear in the description of the next entry and other stuff like that. I'll do some tests soon with it and report the results back here.

**Reply by fernandocamargoai** [reply]
Using the `classic` theme, Indeed mixes up the highlight lines:
![Screenshot_20240305_190045](https://github.com/sinaatalay/rendercv/assets/3669312/366786bd-43e6-4259-a713-eb7cb9f0e0a2)

**Reply by fernandocamargoai** [reply]
For `moderncv`, the parsing works fine for work experiences that contain highlights (even though I miss having clear list markers), but it gets completely lost with entries with no highlights.
